packages/as-types/assembly/serialization/arrays.ts

```diff
@@ -0,0 +1,22 @@
+// @ts-ignore: decorator
+@inline @lazy const MIN_SIZE: usize = 8;
```

Comment:
Why 8 ?

Reply:
I copied this from assemblyscript array

Reply:
weird :/

Reply:
this seems to work just fine (with your unit tests)
```
export function arrayToBytes<T>(array: T[]): StaticArray<u8> {
  return changetype<StaticArray<u8>>(array);
}

export function bytesToArray<T>(array: StaticArray<u8>): T[] {
  return changetype<T[]>(array);
}

Reply:
Yes, but the problem here is that both have the same pointer...